packages/climate/climate.0.1.0/opam

```diff
@@ -0,0 +1,36 @@
+# This file is generated by dune, edit dune-project instead
+opam-version: "2.0"
+synopsis: "Declarative command-line parser for OCaml"
+maintainer: ["https://github.com/gridbugs/climate/issues"]
+authors: ["Stephen Sherratt"]
+license: "MIT"
+homepage: "https://github.com/gridbugs/climate"
+bug-reports: "https://github.com/gridbugs/climate/issues"
+depends: [
+  "dune" {>= "3.0"}
+  "ocaml"
```

Comment:
```suggestion
  "ocaml" {>= "4.14"}
```

The CI fails on earlier versions of OCaml (syntax error, `Strings.starts_with`, `String.cat`, and `Seq.init`). If you want to have a wider set of compatible versions, you could look into not using some of those functions upstream. But for packaging purposes, the version constraint is what's needed.

Reply:
Done!